src/bicep/modules/defender.bicep

```diff
@@ -5,7 +5,9 @@ Licensed under the MIT License.
 
 targetScope = 'subscription'
 
-var bundle = (environment().name != 'AzureUSGovernment' ? [  
+param envName string = environment().name
```

Comment:
We don't expect users to input their own environment names here, do we? I think this should be removed entirely, or if we want to assign it to a `var` instead for readability, I could understand that.

Reply:
> Thanks for this PR. ✨ The empty array looks good. Did you discover that it's not possible to retrieve the list of bundles during the deployment?
> 
> I'm running a test deployment but here are my initial comments.

Thanks @brooke-hamilton 
In answer to your question, I was not able to find a way to retrieve the bundle values automatically and implement them via bicep

Reply:
@glennmusa will change it to be more in line with what was implemented prior; this variable may not be needed but the parameter environment().name is needed. Fixing this now.

Reply:
Updated in commit: 
[e3791a3](https://github.com/Azure/missionlz/pull/687/commits/e3791a317900377d4967feddccb7ecb6b357a1c3)

Reply:
I'm cool with how source looks, will leave review up to @brooke-hamilton's deployment test and doc review 👍